doc/source/user_guide/merging.rst

```diff
@@ -206,10 +206,9 @@ behavior:
 
 .. warning::
 
-   The default behavior with ``join='outer'`` is to sort the other axis
-   (columns in this case). In a future version of pandas, the default will
-   be to not sort. We specified ``sort=False`` to opt in to the new
-   behavior now.
+    .. versionchanged:::::: 1.0.0
```

Comment:
this *should* be in the doc-string is it not?

Reply:
It is, but ths warning previously mentioned, that this behavior will change in a future version, which was wrong because it was changed in 1.0. So we should either add, that it has already changed or remove it altogether. Based on @jorisvandenbossche recommendation I have removed the warning.